links.json

```diff
@@ -186,8 +241,13 @@
         },
         {
           "title": "Genspark AI",
-          "url": "https://www.genspark.ai",
+          "url": "https://genspark.ai",
```

Comment:
Is there any specific reason to shift from "https://www.genspark.ai" to "https://genspark.ai"